src/main/java/nl/knaw/dans/migration/core/VaultLoader.java

```diff
@@ -86,15 +86,21 @@ public void deleteBatch(String batch, String bagStore) {
 
   @UnitOfWork("hibernate")
   public void loadFromVault(UUID uuid, Mode mode, String batch, String bagStore) {
-    final BagInfo bagInfo = bagInfoFromIndex(uuid.toString());
+    BagInfo bagInfo;
+    try {
+      bagInfo = bagInfoFromIndex(uuid.toString());
+    } catch ( Exception e) {
+      inputDatasetDAO.create(new InputDataset(uuid, "SKIPPED " + e, batch, bagStore));
```

Comment:
This is not load-from-fedora. I guess when verify-migration can't find a uuid in the bag-index, convert-bag-to-deposit most likely also could not unless the bag-index was down. 

Looking at this, line 105 should move down, once in the then branch, once in the for-loop of the else-branch.

Reply:
Sorry, now I am mistaken about line 105. The members of a bag-sequence are listed in the uuid_v2 column.

Reply:
my confusion was on the name `SKIPPED`, which will be recorded in the input_datasets. But this has a different meaning than `SKIPPED` in the input-csv. I think this will be cause for head scratching in the future. For processed records, the value of the comment column is taken from the input-csv. So `SKIPPED` can then have multiple meanings.

Reply:
How would you like to resolve? Drop the word skipped and just hang on to the exception message?

Reply:
Similar question for line 98 and 100.

Reply:
my initial idea was to have the input_datasets reflect the datasets we have offered to easy-fedora-to-bag or retrieved from the vault. So also when a dataset is SKIPPED or FAILED by easy-fedora-to-bag it should show that in the input_datasets table. That way, we have an overview of all selected datasets for a migration, and in what batch they were processed.
In that light, i would not expect this error message there at all. The comment in this table is about the export process, not about the verification process